src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -801,10 +803,46 @@ def download_image(self, iso_download_path: str = None) -> Path:
             )
         return self._infra_env.download_image(iso_download_path)
 
+    # Downloads the iPXE script from assisted service to a local file
```

Comment:
add it as a docstring. e.g.:
```
def prepare_local_ipxe_server(...):
    """Downloads the iPXE script from..."""
    assisted_service_url = ...
```

Reply:
Thanks for showing me the syntax and proper way to document!